easybuild/easyconfigs/b/BOINC/BOINC-7.16.5-foss-2019b-client.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'BOINC'
+version = '7.16.5'
+versionsuffix = "-client"
+
+homepage = 'https://boinc.berkeley.edu'
+description = """BOINC is a program that lets you donate your idle computer time to science projects
+ like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/BOINC',
+        'repo_name': 'boinc',
+        'tag': 'client_release/%(version_major_minor)s/%(version)s',
+    },
+}]
+checksums = ['d8c808682d0549b7e1045f1b4633aeb8287619014effc4c16c7b4145b089f988']
+
+dependencies = [
+]
+
+builddependencies = [
+]
```

Comment:
Why remove all the builddeps? Definitely looks like at least some of them are needed.

Reply:
How do I work out what is needed?  Should, say, 'make' be specified even though the easyblock is ConfigureMake?

Reply:
Autotools based projects should normally provide a pre-generated configure and Makefile (which leads to the classic `./configure; make; make install` procedure). But, for generating the scripts to start (which it looks like `./_autosetup` is doing), you need all the Autotools stuff.

For `make` itself, i must admit I'm not sure.

Reply:
Added Autotools.

Reply:
Just in case you missed it ; you haven't actually updated the PR

Reply:
Doesn't pushing to the PR branch update it? Or are you referring to the fact that I resolved the conversation before actually pushing the commit?

Reply:
It does, it was just that i didn't see any new changes even an hour after the comment, so i thought you might have missed it.